pages/windows/dir.md

```diff
@@ -0,0 +1,19 @@
+# dir
+
+> List directory contents.
+
+- Show contents of current directory:
+
+`dir`
+
+- Show contents of a given directory:
+
+`dir {{path/to/directory}}`
+
+- Show hidden contents of current directory:
```

Comment:
Just saying "Show hidden contents" might give the impression that it shows _only_ hidden contents. How about "Show contents of current directory, including hidden files" ?

Reply:
It only shows the hidden content

Reply:
In that case, just do /A. Because I think generally people will always expect to see the normal files.

Reply:
done.